variables.tf

```
@@ -328,3 +328,11 @@ variable "bucket_key_enabled" {
   For more information, see: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-key.html
   EOT
 }
+
+variable "enable_cloud_trail_bucket_policy" {
```

see other input variable names to be consistent with the naming convention i.e.

```suggestion
variable "cloudtrail_bucket_policy_enabled" {
```